content/vault/v1.21.x (rc)/content/docs/commands/secrets/tune.mdx

```diff
@@ -49,6 +52,15 @@ Specify multiple audit non-hmac request keys:
 $ vault secrets tune -audit-non-hmac-request-keys=common_name -audit-non-hmac-request-keys=ttl pki/
 ```
 
+</Tab>
+
+<Tab heading="GUI" group="gui">
+
```

Comment:
```suggestion

@include 'alerts/secret-engine-tune-gui-support.mdx'

```

Reply:
Make sure to include the extra whitespace so the markdown parser doesn't get confused 😅